app/models/mixins/retirement_mixin.rb

```diff
@@ -10,9 +10,9 @@ module RetirementMixin
   end
 
   module ClassMethods
-    def make_retire_request(*src_ids, requester)
+    def make_retire_request(*src_ids, requester, initiator: 'user')
```

Comment:
I think we should do some sort of validation that `initiator` is `"user"` or `"system"` or any other acceptable values.  We don't want someone accidentally sticking a `User` object in here.